pthread-makefile-examples/2_pthread_join/Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile Tutorial 2: Macro, PHONY Targets 	
+
+#					Macros
+# The make program allows you to use macros, which are similar to variables. 
+# Macros are defined in a Makefile as = pairs. 
+# An example has been shown below −
```

Comment:
@DeekshithPatil can you link some references here, maybe just relevant section of the gnu make manual?